### PR TITLE
Update RNA-Seq README.md

### PR DESCRIPTION
**Pull request by jmsdao**
Added some extra expectations to the requirements to we don't repeat the same points in reviews over and over.

### CODE REVIEW

File: RNA-Seq Analysis/README.md [hunk 1]
```diff
@@ -12,12 +12,17 @@ Find an RNA-Seq dataset and perform differential gene expression analysis to det
...
 
 Then, do the following in R Markdown:
-1. Use DESeq2 for the differential gene expression analysis. Find the DEGs between your conditions of interest. 
+1. Use DESeq2 for the differential gene expression analysis. Find the DEGs between your conditions of interest.
+	- Make sure to add an interactable table of DEGs using a package such as [DT](https://rstudio.github.io/DT/)
 2. Create a PCA plot colored by the condition of interest
 3. Create a Volcano Plot
-4. Create a heatmap showing the top 20 over- and under-expressed DEGs
+4. Create a heatmap with the top 10 over-expressed DEGs, and top 10 under-expressed DEGs
+	- Top is defined by DEGs having the lowest adjusted p-values, where the absolute log2 fold change is > 1
```
Comment (millerh1) [inline RNA-Seq Analysis/README.md]: I think we should only look at padj as log2 fold change (without shrinkage estimation) is misleading in many cases

File: RNA-Seq Analysis/README.md [hunk 2]
```diff
@@ -12,12 +12,17 @@ Find an RNA-Seq dataset and perform differential gene expression analysis to det
...
 5. Do GSEA on the results and plot the top 5 pathways
+	- Make sure you use an appropriate [ranking metric](https://www.gsea-msigdb.org/gsea/doc/GSEAUserGuideTEXT.htm#_Metrics_for_Ranking) (other metrics outside this document are also acceptable if justified)
```
Comment (millerh1) [inline RNA-Seq Analysis/README.md]: I think we should probably explicitely tell them to use the wald `stat` column for this

**Review by millerh1 — CHANGES_REQUESTED** [review]
Hi @jmsdao -- thanks for adding this! Would you mind making the changes requested?

**Comment by jmsdao** [comment]
@millerh1 I just pushed the changes requested. Please review and merge if appropriate

**Review by millerh1 — APPROVED** [review]
Looks good 😄

**Pull request merged**